code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1378,14 +1378,14 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 
 	if(gravity && !flight)	//Check for chemicals and innate speedups and slowdowns if we're on the ground
 		if(HAS_TRAIT(H, TRAIT_GOTTAGOFAST))
-			. -= 1
+			. -= 0.5
 		if(HAS_TRAIT(H, TRAIT_GOTTAGOREALLYFAST))
-			. -= 2
+			. -= 1
 		. += speedmod
 		. += H.physiology.speed_mod
 
 	if (H.m_intent == MOVE_INTENT_WALK && HAS_TRAIT(H, TRAIT_SPEEDY_STEP))
-		. -= 1.5
```

Comment:
This only works well walking as well as costing a lot of points for the trait thats only gettable via the Trait system...